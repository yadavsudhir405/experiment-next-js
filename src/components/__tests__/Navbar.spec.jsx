import React from 'react'
import {render} from "@testing-library/react";

import {Navbar} from "../index";



describe('Navbar', () => {
    it('should match the snapshot', () => {

        const {container} = render(<Navbar />);

        expect(container).toMatchSnapshot();
    })
});
