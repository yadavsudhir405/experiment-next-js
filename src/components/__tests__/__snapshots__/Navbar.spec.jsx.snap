// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match the snapshot 1`] = `
<div>
  <div
    class="jsx-2530531754"
  >
    <ul
      class="jsx-2530531754"
    >
      <li
        class="jsx-2530531754"
      >
        <a
          class="jsx-2530531754"
          href="/"
        >
          Home
        </a>
      </li>
      <li
        class="jsx-2530531754"
      >
        <a
          class="jsx-2530531754"
          href="/about"
        >
           About
        </a>
      </li>
    </ul>
  </div>
</div>
`;
